tests/c_languages/codeclone_detection/conditions_samples.c

```diff
@@ -150,3 +150,30 @@ int structing(struct test_struct a, struct test_struct *b) {
     a.a = 1 + b;
     (b+2)->a = 2;
 }
+
+int isswitch(int a, int b) {
+    switch (b) {
+        case a:
```

Comment:
I think in `case a` the `a` is rather part of the condition than part of the body, your implementation does it the other way round but that doesn't make sense does it? (Am I being stupid?)